pom.xml

```
@@ -55,8 +58,8 @@
 		<dependencies>
 			<dependency>
 				<groupId>io.jenkins.tools.bom</groupId>
-				<artifactId>bom-2.289.x</artifactId>
-				<version>1362.v59f2f3db_80ee</version>
+				<artifactId>bom-weekly</artifactId>
```

https://github.com/jenkinsci/bom?tab=readme-ov-file#depending-on-bom-weekly

```suggestion
				<artifactId>bom-2.462.x</artifactId>
```